src/main/java/bio/terra/service/snapshotbuilder/utils/constants/Person.java

```diff
@@ -9,4 +9,5 @@ private Person() {}
   public static final String GENDER_CONCEPT_ID = "gender_concept_id";
   public static final String RACE_CONCEPT_ID = "race_concept_id";
   public static final String ETHNICITY_CONCEPT_ID = "ethnicity_concept_id";
+  public static final String ROW_ID = "datarepo_row_id";
```

Comment:
This duplicates another constant, `PdaoConstant.PDAO_ROW_ID_COLUMN`. I would use that here, or replace `ROW_ID` with `PDAO_ROW_ID_COLUMN` in this code.